clusterloader2/testing/load/config.yaml

```diff
@@ -1,12 +1,33 @@
+#Constants
+{{$BIG_GROUP_SIZE := 250}}
+{{$MEDIUM_GROUP_SIZE := 30}}
+{{$SMALL_GROUP_SIZE := 5}}
+{{$PODS_PER_NODE := 30}}
+{{$NODES_PER_NAMESPACE := 100}}
+#Variables
+# bigRCs - 1/4 of all pods should be in big RCs.
+{{$bigRCs := MultiplyInt .Nodes 0.03}}
+# mediumRCs - 1/4 of all pods should be in medium RCs.
+{{$mediumRCs := MultiplyInt .Nodes 0.25}}
+# smallRCs - 1/2 of all pods should be in small RCs.
+{{$smallRCs := MultiplyInt .Nodes 3}}
+{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
+{{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
+{{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
+{{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+# saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
+{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPods 10)}}
+#Test
 name: load
-automanagedNamespaces: 3
+automanagedNamespaces: {{$namespaces}}
 tuningSets:
-- name: Average0.33qps
+- name: AvgSaturationQPS
   randomizedLoad:
-    averageQps: 0.33
-- name: Average1.4qps
+    averageQps: {{$saturationQPS}}
+- name: AvgScalingQPS
   randomizedLoad:
-    averageQps: 1.4
+    averageQps: {{MultiplyFloat $saturationQPS 4}}
```

Comment:
Could you add a comment here explaining this factor of 4? (you should be able to ~copy it from our load test)

Reply:
Comment added.